ReactWindows/ReactNative.Net46.Tests/Views/Scroll/ReactScrollViewManagerTests.cs

```diff
@@ -0,0 +1,193 @@
+﻿using NMock;
+using NUnit.Framework;
+using ReactNative.Views.Scroll;
+using System;
+using System.Threading.Tasks;
+
+namespace ReactNative.Tests.Views.Scroll
+{
+    [TestFixture]
+    class ReactScrollViewManagerTests
+    {
+        private MockFactory _mockFactory;
+        private TestScrollView _testScrollView;
+        private Mock<IScrollViewScroller> _mockScrollViewScroller;
+        private ReactScrollViewManager _scrollViewManager;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockFactory = new MockFactory();
+            _mockScrollViewScroller = _mockFactory.CreateMock<IScrollViewScroller>();
+            _testScrollView = new TestScrollView(_mockScrollViewScroller.MockObject);
+            _scrollViewManager = new ReactScrollViewManager();
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            _mockFactory.VerifyAllExpectationsHaveBeenMet();
```

Comment:
I think you want to do this in each test, in the "assert" section, or even better by verifying individual mocks immediately after their expectations are met. This makes the timeline of interactions more explicit in the tests, and also makes for better failure reporting by the test runner.